textdb/textdb-api/src/main/java/edu/uci/ics/textdb/api/common/FieldType.java

```diff
@@ -10,5 +10,8 @@
 	// A field that is indexed and tokenized,without term vectors
 	TEXT,
 	//A field that is the list of values
-	LIST;
+	LIST,
+	//A field that is generic, and allow specific operations like Term Vectors

```

Comment:
I don't quite follow this comment about "GENERIC_FIELD". Also the grammar of the comment needs to be fixed.